common/src/main/java/io/netty/util/MacAddress.java

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright 2016 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.util;
```

Comment:
Strictly speaking "internal" is not part of the public API, and intended to be a spot where we can share code across packages to use internal to Netty. However these classes are typically publicly visible so there is nothing stopping folks from using them, if they are willing to assume the risks of using internal packages. My hesitation to expanding the public API for what seems like something that is not a core part of using Netty (and may be based on a heuristic). @normanmaurer - WDYT?